### PR TITLE
feat(integrations): add streamelements integration

### PR DESCRIPTION
**Pull request by Banonevar**
I just started. I'm waiting for fix that issue https://github.com/sogehige/sogeBot/issues/2699 for continue developing this integration.

###### CHECKLIST

- [ ] tests are changed or added
- [ ] documentation is changed or added
- [x] locales are changed or added
- [ ] db relationship (tools/database) are changed or added
- [ ] commit message follows [commit guidelines](https://github.com/sogehige/sogeBot/blob/master/CONTRIBUTING.md#commit-guidelines)

### CODE REVIEW

**Comment by Banonevar** [comment]
I think i'm done with that, but still it not properly tested.

File: locales/cs/ui/integrations/streamelements.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "settings": {
+    "enabled": "Status",
+    "jwtToken": {
+      "title": "JWT token",
+      "help": "Get JWT token at dashboard settings->click show secrets"
```
Comment (sogehige) [inline locales/cs/ui/integrations/streamelements.json]: JWT token lze nalézt v StreamElements dashboardu, settings->click show secrets

File: locales/en/ui/integrations/streamelements.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "settings": {
+    "enabled": "Status",
+    "jwtToken": {
+      "title": "JWT token",
+      "help": "Get JWT token at dashboard settings->click show secrets"
```
Comment (sogehige) [inline locales/en/ui/integrations/streamelements.json]: Get JWT token at **StreamElements** dashboard settings->click show secrets

File: src/bot/integrations/streamelements.ts [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      message: eventData.data.message,
+      autohost: false,
+    });
+
+    triggerInterfaceOnTip({
+      username: eventData.data.username.toLowerCase(),
+      amount: eventData.data.amount,
+      message: eventData.data.message,
+      currency: eventData.data.currency,
+      timestamp: _.now(),
```
Comment (sogehige) [inline src/bot/integrations/streamelements.ts]: _.now() => Date.now()

File: src/bot/integrations/streamelements.ts [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    }
+    global.overlays.eventlist.add({
+      type: 'tip',
+      amount: eventData.data.amount,
+      currency: eventData.data.currency,
+      username: eventData.data.username.toLowerCase(),
+      message: eventData.data.message,
+      timestamp: Date.now(),
+    });
+    tip(`${eventData.data.username.toLowerCase()}, amount: ${eventData.data.amount}${eventData.data.currency}, message: ${eventData.data.message}`);
```
Comment (sogehige) [inline src/bot/integrations/streamelements.ts]: you are using a lot of eventData.data.username.toLowerCase() => make a variable username

Comment (sogehige) [inline src/bot/integrations/streamelements.ts]: add id of user please
like ${username}#${id}

**Pull request merged**